lib/ExtUtils/ParseXS.pm

```diff
@@ -2144,6 +2143,39 @@ sub eval_input_typemap_code {
   return ExtUtils::ParseXS::Eval::eval_input_typemap_code($self, $code, $other);
 }
 
+# Split arguments from C/C++ function signatures
+sub C_CPP_split_arguments {
+  my $function_arguments = shift;
+  if ($function_arguments =~ m/\<.*,.*\>/) {
+    # More than one template argument
+    my @temp_args = split(/\s*,\s*/, $function_arguments);
+    my @args;
+    my $argument;
+    for (my $i = 0; $i < scalar @temp_args; $i++) { 
+        $argument = $temp_args[$i];
+        if ($temp_args[$i] =~ m/\</) {
+            my $number_of_opening_templates = () = ($temp_args[$i] =~ /\</g);
```

Comment:
Yes it is! Changed via https://github.com/Perl-Toolchain-Gang/extutils-parsexs/pull/3/commits/2ee95a387184fde98a858928fe432441407f256e